packages/react-native-bridge/android/build.gradle

```diff
@@ -114,6 +120,11 @@ android {
     sourceSets {
         main {
             assets.srcDirs += buildAssetsFolder
+            assets.srcDirs += '../../../../src/block-support'
+            // Despite being in a folder called "resources", the files in
+            // unsupported-block-editor are accessed as assets by their
+            // consumers: the WordPressEditor library.
+            assets.srcDirs += '../../../../resources/unsupported-block-editor'
```

Comment:
These are not _required_ for the AAR generation, but we'll use them to address https://github.com/wordpress-mobile/WordPress-Android/issues/13747.

Reply:
Right, I assume this is something we'll take another look at before merging to `master`.